starter-code/apple-pie-starter-code/css/main.css

```diff
@@ -0,0 +1,48 @@
+
+*{
+    font-family:'Open Sans', Arial, Helvetica, sans-serif;
+}
+
+/* -----------------------*/
+
+.Content{
```

Comment:
be careful on using capital letters, i leave you some guide about naming good practices in css:

http://bdavidxyz.com/blog/how-to-name-css-classes/